cell_classification/segmentation_data_prep_test.py

```diff
@@ -122,15 +123,34 @@ def test_check_input():
         conversion_matrix_path = os.path.join(temp_dir, "conversion_matrix.csv")
         conversion_matrix.to_csv(conversion_matrix_path, index=False)
         norm_dict = {"CD11c": 1.0, "CD14": 1.0, "CD56": 1.0, "CD57": 1.0}
-        data_folders = prepare_test_data_folders(5, temp_dir, norm_dict.keys())
+        with open(os.path.join(temp_dir, "norm_dict.json"), "w") as f:
+            json.dump(norm_dict, f)
+        data_folders = prepare_test_data_folders(5, temp_dir, list(norm_dict.keys()) + ["XYZ"])
         cell_table_path = os.path.join(temp_dir, "cell_type_table.csv")
         cell_table = prepare_cell_type_table()
         cell_table.to_csv(cell_table_path, index=False)
 
+        # CONVERSION MATRIX
+        # check if conversion_matrix is loaded correctly in check_input
```

Comment:
The purpose of the check_input function is to find any errors in the parameters the user provided. Since we're not relying on this function to append the conversion matrix as attribute or anything, I don't think it's necessary to have a correctness check here.